Assets/HoloToolkit-Examples/Boundary/Materials/Highlight.mat

```diff
@@ -7,8 +7,8 @@ Material:
   m_PrefabParentObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
   m_Name: Highlight
-  m_Shader: {fileID: 4800000, guid: f326278a838e87c499393f30a8e72127, type: 3}
-  m_ShaderKeywords: ETC1_EXTERNAL_ALPHA _EMISSION
+  m_Shader: {fileID: 4800000, guid: 06790e211e4a6df409b1e04bc91c4810, type: 3}
```

Comment:
What's the new shader we're using?

Reply:
Unlit Configurable. The old guid is missing, and the material had the magenta error shader applied.